src/routes/safe/components/Transactions/TxList/TxCollapsedActions.tsx

```diff
@@ -36,7 +36,7 @@ export const TxCollapsedActions = ({ transaction }: TxCollapsedActionsProps): Re
   const txStatus = useLocalTxStatus(transaction)
 
   const getTitle = () => {
-    if (txStatus === LocalTransactionStatus.AWAITING_EXECUTION) {
+    if (txStatus === LocalTransactionStatus.AWAITING_EXECUTION || LocalTransactionStatus.PENDING_FAILED) {
```

Comment:
Did you merge the other PR? Why is this included here?

Reply:
I had implemented the fixes here before opening the other PR. Going to remove it